src/map/mob.cpp

```diff
@@ -4811,6 +4812,18 @@ uint64 MobDatabase::parseBodyNode(const ryml::NodeRef& node) {
 		mob->status.amotion = cap_value(speed, battle_config.monster_max_aspd, 2000);
 	}
 
+	if (this->nodeExists(node, "ClientAttackMotion")) {
+		uint16 speed;
+
+		if (!this->asUInt16(node, "ClientAttackMotion", speed))
+			return 0;
+
+		mob->status.clientamotion = cap_value(speed, 1, USHRT_MAX);
+	} else {
+		if (!exists)
+			mob->status.clientamotion = cap_value(mob->status.amotion, 1, USHRT_MAX);
```

Comment:
```suggestion
			mob->status.clientamotion = cap_value(mob->status.amotion, 1, 432);
```
Or a newly introduced constant.

Reply:
Needs to stay capped to USHRT_MAX or it won't work. There are plenty of official values higher than 432.